TicTacToe/Logic/PlayerMove.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace TicTacToe.Logic
+{
+    public class PlayerMove
+    {
+        public PlayerMove(int subBoardRow, int subBoardCol, int atomicCellRow, int atomicCellCol, PlayerMarker playerMarker)
+        {
+            SubBoardRow = subBoardRow;
+            SubBoardCol = subBoardCol;
+            AtomicCellRow = atomicCellRow;
+            AtomicCellCol = atomicCellCol;
+            PlayerMarker = playerMarker;
+        }
+
+        public int SubBoardRow { get; }
```

Comment:
Now where I get reminded of these properties, I think the type "AtomicCellId" suggested earlier should in fact be called "BoardCellId" since it's fit for more than just atomic cells